docs/framework/guides/document-editor.md

```diff
@@ -183,28 +191,30 @@ Then focus on headings and paragraphs. Note that what the users see in the headi
 	margin-bottom: .142em;
 }
 
-/* Set the styles for "Heading 2". */
+/* Set the styles for the "Heading 2". */
```

Comment:
It's actually OK without "the", since it's written as a proper name, with a capital letter. The same below.